webapps/camunda-commons-ui/grunt/tasks/ensureLibs.js

```diff
@@ -17,13 +17,13 @@
 
 var fs = require('fs');
 var through = require('through2');
+const path = require('path');
 
 var included = [
```

Comment:
Note: `fast-xml-parser` isn't in the `included` list. Is this intended?

Reply:
Nice catch! Not really intended. I just overlooked it because it is already included in the `deps.js`. Probably because it comes transitively with another library that is already included and then automatically pulled in by browserify.